ui/src/views/storage/UploadLocalVolume.vue

```diff
@@ -103,12 +88,7 @@
           </a-select>
         </a-form-item>
         <a-form-item>
-          <span slot="label">
-            {{ $t('label.volumechecksum') }}
-            <a-tooltip :title="apiParams.checksum.description">
-              <a-icon type="info-circle" style="color: rgba(0,0,0,.45)" />
-            </a-tooltip>
-          </span>
+          <tooltip-label slot="label" :title="$t('label.volumechecksum')" :tooltip="apiParams.volumechecksum.description"/>
```

Comment:
```suggestion
          <tooltip-label slot="label" :title="$t('label.checksum')" :tooltip="apiParams.checksum.description"/>
```

Reply:
Made it `<tooltip-label slot="label" :title="$t('label.volumechecksum')" :tooltip="apiParams.checksum.description"/>`  with title, tooltip values as they were before refactor